sdk/spring/azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/aad/UserGroup.java

```diff
@@ -6,21 +6,42 @@
 import java.io.Serializable;
 import java.util.Objects;
 
+import com.fasterxml.jackson.annotation.JsonAlias;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
 public class UserGroup implements Serializable {
     private static final long serialVersionUID = 9064197572478554735L;
 
+    @JsonProperty("objectId")
+    @JsonAlias("id")
     private String objectID;
+
+    @JsonProperty("objectType")
+    @JsonAlias("@odata.type")
```

Comment:
When will `id` and `objectType` been used? In the json I get, it only have `objectId` and `odata.type`.

Reply:
`id` and `objectType` are used depending on if the request is to Azure AD Graph API (v1) or the Microsoft Graph API (v2). See example responses here: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot/src/test/resources/aad

For reference, here are the docs for the `/memberOf` request/response for both versions:
**Azure Graph API**: https://docs.microsoft.com/en-us/previous-versions/azure/ad/graph/api/users-operations#get-a-users-group-and-directory-role-memberships--
**Microsoft Graph API:** https://docs.microsoft.com/en-us/graph/api/user-list-memberof?view=graph-rest-1.0&tabs=http